src/main/webapp/js/common.js

```diff
@@ -558,17 +558,13 @@ function scrollToTop(duration) {
 var DIV_STATUS_MESSAGE = '#statusMessagesToUser';
 
 /**
- * Sets a status message and the message status.
+ * Prepares a status message and the message status.
  * Default message type is info.
  *
  * @param message the text message to be shown to the user
  * @param status type
  */
-function setStatusMessage(message, status) {
-    if (message === '' || message === undefined || message === null) {
-        return;
-    }
-
+function prepareStatusMessage(message, status) {
```

Comment:
I think the function name can be improved and made more descriptive. Generally, `prepare...` is only slightly better than `process...`, and that's only when the word `prepare` is used accurately.